apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -281,7 +296,7 @@ class ManageStudentsTable extends Component {
           props: {
             style: {
             ...tableLayoutStyles.cell,
-            width: 300
+            width: 260
```

Comment:
I just changed these so the cog icon won't stack over the text.  We can decide later what the best sizing/layout polish should be.